pettingzoo/sisl/waterworld/waterworld.py

```diff
@@ -71,7 +72,7 @@ def render(self, mode="human"):
         return self.env.render(mode)
 
     def step(self, action):
-        if self.dones[self.agent_selection]:
+        if self.terminations[self.agent_selection]:
```

Comment:
`if terminations or truncations`?

Reply:
I wanted to do terminations or truncations, but this method allows the user to continue stepping the environment even when truncation is true. This is useful for debugging.

Reply:
Turns out tests fail when this happens, so I'm reverting to `terminations or truncations`.